crd-generator/maven-plugin/src/main/java/io/fabric8/crd/generator/maven/plugin/Dependency.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.crd.generator.maven.plugin;
+
+public class Dependency {
```

Comment:
I think I didn't choose this existing model because it allows own filtering and I tried to avoid filtering at too many levels.

I currently investigate in other ways of filtering, mainly because of your other comment (https://github.com/fabric8io/kubernetes-client/pull/5979#discussion_r1717034119).

If I can find a way to improve it, should I push it to this branch or is the current state good enough to get merged?

Reply:
This branch is good to go, let's leave it as is. I'll merge it on Monday (I'm leaving some time in case someone else wants to review).